package.json

```diff
@@ -59,6 +59,7 @@
     "@plotly/d3-sankey": "^0.5.0",
     "alpha-shape": "^1.0.0",
     "array-range": "^1.0.1",
+    "base64-arraybuffer": "^0.1.5",
```

Comment:
https://github.com/niklasvh/base64-arraybuffer/blob/master/lib/base64-arraybuffer.js seems like a legit and fast implmentation. Nice find!

Now, we'll have to check-in the associated `package-lock.json` change, to get CI to build properly.

Reply:
Done in 5030d3a